recipes/Open3D/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set name = "Open3D" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isl-org/Open3D/archive/v{{ version }}.tar.gz
+  sha256: 524ddeb7dc8aaed6dc5b272415df0c8ffcf7eff0816a84cfbdee56cdd6d3587a
+  patches:
+    - fix-tbb.patch
+    - fix-unzip.patch
+    - fix-curl.patch
+    - fix-error.patch
+    - fix-find-imgui.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  ignore_run_exports_from:
+    - libboost-devel
```

Comment:
```
Could NOT find Boost: Found unsuitable version "1.82.0", but required is exact version "1.84.0" (found $PREFIX/include, )
```
Your package requires a specific version of boost, and it's not getting that version. Maybe that's why you have to ignore run exports?

Reply:
This is an issue in vtk (see next line "Could not find the VTK package due to a missing dependency: Boost"). The vtk recipe uses `libboost-headers` only (version 1.84.0, as there is no global pin on libboost-headers), but then their cmake scripts require the exact version of boost when projects depend on vtk.

I will try again just depending on `libboost-headers` and see how we go, but I think I've encountered issues with that previously.

Reply:
`libboost-headers` works just fine :)